### PR TITLE
[#31] Create scss folder structure

### PR DESCRIPTION
**Pull request by kammanz**
Rudimentary scss module structure in place. I haven't changed any class names to camel case because nothing is broken, and from what I've read, there are only some cases (pardon pun) where it will break. Having said that, if you want me to change it I will. Regarding color names, you said we would talk about it at a later point.

### CODE REVIEW

**Comment by Tumne** [comment]
If you can please change the base branch from `master` to `RAP-27c__Selected-and-hover-players-card` and add a `Branched PR` label on this.

File: src/components/Header.js [hunk 1]
```diff
@@ -6,13 +6,17 @@ import bell from '../assets/icons/notification-bell.svg';
 import settingsIcon from '../assets/icons/settings-icon.svg';
 import kobe from '../assets/imgs/kobe.png';
 
+import styles from './header.module.scss';
+
 const Header = () => {
+    console.log(styles, 'styles');
```
Comment (Tumne) [inline src/components/Header.js]: remove console.log

File: src/components/Header.js [hunk 2]
```diff
@@ -6,13 +6,17 @@ import bell from '../assets/icons/notification-bell.svg';
...
+
     return (
         <div className="header-container">
             <div className="team">
                 <button className="logo-container">
                     <img src={raptorsLogo} title="raptors logo" alt="raptors logo"></img>
-                    <span>Toronto Raptors </span>
+                    <span className={styles.redStuff}>Toronto Raptors </span>
```
Comment (Tumne) [inline src/components/Header.js]: I don't see any styles associated with `.redStuff`. (An aside, try to be lil' more formal when renaming items "redStuff" lol) Eitherway, let's remove this className attribute

File: src/components/filters.module.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@import '../scss/variables.scss';
```
Comment (Tumne) [inline src/components/filters.module.scss]: use absolute imports instead

`@import 'src/scss/variables.scss';`

File: src/components/header.module.scss [hunk 4]
```diff
@@ -0,0 +1,138 @@
+// Header Styles
+
```
Comment (Tumne) [inline src/components/header.module.scss:2]: remove comment and space

File: src/components/header.module.scss [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+@import '../scss/variables.scss';
```
Comment (Tumne) [inline src/components/header.module.scss]: Use absolute import

File: src/components/players/playersList.module.scss [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+.players-list-container {
+    display: flex;
+    flex-wrap: wrap;
+    width: 1044px;
+    justify-content: space-between;
+    background-color: $white;
+}
+  
+
+// Player Card Styles
```
Comment (Tumne) [inline src/components/players/playersList.module.scss]: remove unnecessary comment

File: src/scss/App.scss [hunk 7]
```diff
@@ -1,31 +1,9 @@
...
-$background-color-grey: #EDEDED;
-$text-dark-grey: #919191;
-$border-shadow: #D4D3D3;
-$border-dark-grey: #919191;
-$grey-three: #919191;
-$white: #FFFFFF;
+@import '../components/header.module.scss';
+@import '../components/filters.module.scss';
+@import '../components/players/playersList.module.scss';
+@import '../components/players/playerDetails.module.scss';
```
Comment (Tumne) [inline src/scss/App.scss]: Can you add a `TO DO` comment here. Just a reminder that these imports are temporary.

**Review by Tumne — CHANGES_REQUESTED** [review]
Minor changes, all in all looks good. 

If you can make a refactor issue for making the entire app to use css modules that'll be great.

I don't think we should have uppercase for the folder names, I'm going to create another refactor issue for that.

File: src/scss/App.scss (hunk 7 above)
Comment (kammanz) [reply src/scss/App.scss]: I've added the comment. What will eventually happen with these imports, just get deleted?

Comment (Tumne) [reply src/scss/App.scss]: yup, they'll get removed

Comment (Tumne) [reply src/scss/App.scss]: once css modules is completed throughout the app

**Review by Tumne — APPROVED** [review]

**Review by Tumne — CHANGES_REQUESTED** [review]
Let's make everything css modules way. and remove those imports in `App.scss`

File: src/components/players/PlayersList.js [hunk 8]
```diff
@@ -20,36 +22,27 @@ class PlayersList extends React.Component {
...
-                        }
-
-                        if (this.state.selectedCard !== null && el !== this.state.selectedCard) {
-                            this.state.selectedCard.classList.remove("selected-card");
-                            this.setState({selectedCard: el});
-                            el.classList.add("selected-card");
-                        }
-                    
+                    onClick={() => {
+                        this.setState( { selectedId: player.person_id })
```
Comment (Tumne) [inline src/components/players/PlayersList.js]: `this.setState( { selectedId: player.person_id })` should be
`this.setState({ selectedId: player.person_id })`

File: src/scss/App.scss [hunk 9]
```diff
@@ -1,31 +1,4 @@
...
-
-//Player Card Number
-$grey3: #959595;
-
-// Player Card Position
-$grey4: #777777;
-
-// Header Links, Player Card Name
-$grey5: #4e4e4e;
+@import './variables.scss';
```
Comment (Tumne) [inline src/scss/App.scss]: Move `App.scss` file into `components` folder. It should be with App.js

**Review by Tumne — CHANGES_REQUESTED** [review]
Almost there. One more thing we should do now. Let's rename all the js files to be camel case. 
`App.js` becomes `app.js`
`PlayersList.js` becomes `playersList.js`

etc..

This would be a good time to refactor all this and get this out of the way.

**Review by Tumne — COMMENTED** [review]
Also in `playerDetails` line 11, you have a lot of unused variables there.
Going forward make sure not to have unused vars in the code.

**Comment by kammanz** [comment]
Made changes, ready for another PR review.

**Review by Tumne — APPROVED** [review]

**Pull request merged**